ci-operator/step-registry/upi/install/vsphere/upi-install-vsphere-ref.yaml

```diff
@@ -17,6 +17,9 @@ ref:
   - namespace: test-credentials
     name: ci-route-53
     mount_path: /var/run/vault/vsphere-aws
+  env:
+  - name: SECURE_BOOT_ENABLED
+    default: "false"
```

Comment:
Maybe we can add a brief document for the SECURE_BOOT_ENABLED env

Reply:
agree, updated to add documentation part.